### ISSUE TITLE
Compiling with shared libs disabled results in missing lpcap in pkgconfig file

### ISSUE THREAD

**Issue by rockaport** [the-tcpdump-group/libpcap#1009]
This breaks tcpdump as it fails to find libpcap when compiling using cmake. Using autotools/confgure i need to manually update the entry with lpcap for it to work.

```cmake
mkdir build
cd build
cmake -DCMAKE_INSTALL_PREFIX=$OPT -DBUILD_SHARED_LIBS=OFF .. || exit 1
make -j || exit 1
make install || exit 1
```

libpcap.pc
```
prefix="//opt/linux"
exec_prefix="${prefix}"
includedir="${prefix}/include"
libdir="${exec_prefix}/lib"

Name: libpcap
Description: Platform-independent network traffic capture library
Version: 1.11.0-PRE-GIT
Libs: -L${libdir} -l
Libs.private:  -libverbs -lnl-3 -l/usr/lib/x86_64-linux-gnu/libssl.so -l/usr/lib/x86_64-linux-gnu/libcrypto.so
Cflags: -I${includedir}
```

**Comment by guyharris** [comment]
> Using autotools/confgure

Presumably meaning using autotools/configure to build *tcpdump*; you're not using it to build *libpcap*, you're using CMake.

**Comment by rockaport** [comment]
> > Using autotools/confgure
> 
> Presumably meaning using autotools/configure to build _tcpdump_; you're not using it to build _libpcap_, you're using CMake.

I'm using cmake to build libpcap then using autotools to build tcpdump.

**Comment by guyharris** [comment]
Fixed in the trunk and the 1.10 branch.
